### PR TITLE
DTMF support for Android 

### PR DESCRIPTION
**Pull request by brittBarak**
* added a guide on DTMF support. 
* updated the new sdk version on the setup guide.

### CODE REVIEW

File: _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Send and Receive DTMF
+description: This topic shows you how to send and receive DTMF in your app.
+navigation_weight: 3
+---
+
+# Send and Receive DTMF
+
+> **NOTE**: This fueature is currently supported only on Nexmo Client Android SDK.
```
Comment (pardel) [inline _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md]: fueature => feature

File: _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+DTMF is used both for dialing a destination on a landline telephone and also for signalling to the remote end once a call is answered. Typically this is used to implement an Interactive Voice Response (IVR) system, or to enter information like a PIN number or conference call pin.
+
+With Nexmo Client SDKs you can both collect a DTMF input from your app user, and listen to DTMF input that was sent from another member.
+
+## Send DTMF
+
+In order to send DTMF tones from your application to the backend, use this method:
+
+```java
+NexmoCall.sendDTMF(dtmf)
```
Comment (pardel) [inline _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md]: Is `dtmf` made up of one character only or can multiple inputs be sent in one go?

File: _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+The DTMF events will be received in your implementation of `NexmoCallEventListener.onDTMF()` method, on the `NexmoCallEventListener` that is attached to the `NexmoCall`.
+
+For example:
+
+```java
+
+callEventListener = object : NexmoCallEventListener {
+    //…
+
+    override fun onDTMF(digit: String, member: NexmoCallMember) {
```
Comment (pardel) [inline _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md]: Is the listener called once per character?

File: _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md (hunk 2 above)
Comment (brittBarak) [reply _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md]: it's a String. yes I can add a type!

File: _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md (hunk 3 above)
Comment (brittBarak) [reply _documentation/client-sdk/in-app-voice/guides/send-and-receive-dtmf.md]: depending on the events we get from CS, supposed to be all together. changed digit to dtmf. Thank you!

**Pull request merged**